static/js/place/dev_place_main.tsx

```diff
@@ -395,24 +395,46 @@ const RelatedPlaces = (props: {
   place: NamedTypedPlace;
   childPlaces: NamedTypedPlace[];
 }) => {
+  const [isCollapsed, setIsCollapsed] = useState(true);
   const { place, childPlaces } = props;
   if (!childPlaces || childPlaces.length === 0) {
     return null;
   }
+  const NUM_PLACES = 15;
```

Comment:
curious - how did we choose 15?

Reply:
Admittedly a bit arbitrarily -- this looks good to my eyes for now. My thought is to use `NUM_PLACES = 15` to unblock experimental release before the code freeze, and we can decide on a better solution after.

Reply:
After thinking about this more, i think a better experience might be to make this responsive and to show more items when the page is larger, and fewer for smaller screen sizes. Wdyt @miss-o-soup ?

Reply:
Sounds good, I'll discuss with @miss-o-soup and add responsive behavior in a follow-up PR, I don't think this needs to block release.